java/lang/security/audit/tainted-session-from-http-request.yaml

```diff
@@ -0,0 +1,50 @@
+rules:
+  - id: tainted-session-from-http-request 
+    message: >-
+      Detected input from a HTTPServletRequest going into a session command.  
```

Comment:
I'm not certain I follow what this rule detects. What kind of object comes back from request.getSession()? Is it like a mapping of user sessions? What can I do with that session?

Reply:
It's for the "trust boundary violation" portion of the OWASP java benchmark -- there's functions for getSession such as `setAttribute` that sets a parameter for that session (and that's where the issue lies). I'll add that to the message for clarity.